xwiki-platform-core/xwiki-platform-store/xwiki-platform-store-filesystem-oldcore/src/main/java/org/xwiki/store/legacy/doc/internal/FilesystemAttachmentContent.java

```diff
@@ -85,9 +85,17 @@ public FilesystemAttachmentContent(FilesystemAttachmentContent filesystemAttachm
     }
 
     @Override
-    public FilesystemAttachmentContent clone()
+    public XWikiAttachmentContent clone(boolean skipContent)
```

Comment:
The fact I need to override this method and that the overridden clone is not enough anymore shows a breaking change... I don't know if it's ok here.